### ISSUE TITLE
Grid's pager does not pass WCAG validation with AChecker.

### ISSUE THREAD

**Issue by MilenaCh** [telerik/kendo-ui-core#3575]
AChecker requires **_label for_** for select element of Grid's pager to pass WCAG validation.
W3C rule - https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H44

### Reproduction of the problem
1. Navigate to [this ](http://dojo.telerik.com/OCaBi)page

2.Copy the html output of the grid and paste it to [AChecker](https://achecker.ca/checker/index.php).

### Current behavior
There is an [error ](https://www.screencast.com/t/oTFAEHiVJ1ah)about a missing label for select element in the pager.

**Comment by GaloisGirl** [comment]
The select does have an aria-label. 

I think it's an old issue with AChecker and can be closed.

**Comment by gdenchevprog** [comment]
Reported in 1547477, raising priority.

The page number dropdown element which is rendered when the Grid is displayed on a smaller screen is missing a label attribute.

Tested in the following FS Dojo:
https://runner.telerik.io/fullscreen/@gdenchev/EPAFeGoR

![image](https://user-images.githubusercontent.com/70957719/147545835-a01ec867-0d12-4381-83a2-55d6edc20c88.png)
